asyncapi/quickbooks/data_types.bal

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
`WSO2 Inc` -> `WSO2 LLC`

This has to be done in all the licence headers related to quickbooks